lib/cli.ex

```diff
@@ -1,10 +1,33 @@
 defmodule TicTacToe.CLI do
+  alias TicTacToe.Board
 
   def welcome() do
     IO.puts "Welcome to Tic Tac Toe!"
   end
 
+  def print(message) do
+    IO.puts message
+  end
+
+  def display_board(board) do
+    display_spaces = Board.get_display_list(board)
+    IO.write """
+       #{Enum.at(display_spaces, 0)}  |  #{Enum.at(display_spaces, 1)}  |  #{Enum.at(display_spaces, 2)}
+      ----+-----+----
+       #{Enum.at(display_spaces, 3)}  |  #{Enum.at(display_spaces, 4)}  |  #{Enum.at(display_spaces, 5)}
+      ----+-----+----
+       #{Enum.at(display_spaces, 6)}  |  #{Enum.at(display_spaces, 7)}  |  #{Enum.at(display_spaces, 8)}
```

Comment:
I like that you've abstracted away much of the structure of the board by having this method call Board to return a display list that you can then use to format each space.